keps/sig-network/0011-20180518-coredns-default-proposal.md

```diff
@@ -0,0 +1,88 @@
+---
+kep-number: 11
+title: Switch CoreDNS to the default DNS
+authors:
+  - "@johnbelamaric"
+  - "@rajansandeep"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@bowei"
+  - "@thockin"
+approvers:
+  - "@thockin"
+editor: "@rajansandeep"
+creation-date: 2018-05-18
+last-updated: 2018-05-18
+status: provisional
+---
+
+# Switch CoreDNS to the default DNS
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Goals](#goals)
+* [Proposal](#proposal)
+    * [User Cases](#use-cases)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+## Summary
+
+CoreDNS is now well-established in Kubernetes as the DNS service, with CoreDNS starting as an alpha feature from Kubernetes v1.9 to now being GA in v1.11.
+After successfully implementing the road-map defined [here](https://github.com/kubernetes/features/issues/427), CoreDNS is GA in Kubernetes v1.11, which can be installed as an alternate to kube-dns in tools like kubeadm, kops, minikube and kube-up.
+Following the [KEP to graduate CoreDNS to GA](https://github.com/kubernetes/community/pull/1956), the purpose of this proposal is to make CoreDNS as the default DNS for Kubernetes, replacing kube-dns.
+
+## Goals
+* Make CoreDNS the default DNS for Kubernetes for all the remaining install tools (kube-up, kops, minikube).
+* Make CoreDNS available as an image in a Kubernetes repository (To Be Defined) and ensure a workflow/process to update the CoreDNS versions in the future.
+  This goal is carried over from the [previous KEP](https://github.com/kubernetes/community/pull/1956), in case it cannot be completed there.
+
+## Proposal
```

Comment:
do we also want this to encompass the formal deprecation of kube-dns ?

Reply:
ping

Reply:
Yes, I think that makes sense. We don't want to maintain two solutions for any longer than necessary.

Reply:
I guess I was asking it to be in the doc as a goal. :)